examples/network_traffic_prediction/AirTrafficPrediction.ipynb

```diff
@@ -0,0 +1,735 @@
+    "    train_task=False)\n",
+    "test_dataloader = GSgnnNodeDataLoader(\n",
+    "    dataset=ml_data,\n",
+    "    target_idx=ml_data.get_node_test_set(ntypes=['airport']),\n",
+    "    node_feats=nfeats_4_modeling,\n",
+    "    edge_feats=efeats_4_modeling,\n",
+    "    label_field='inventory_amounts',\n",
+    "    fanout=fanout,\n",
+    "    batch_size=64,\n",
+    "    train_task=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "776cd5af-7c0a-4021-8921-23d70d67b488",
+   "metadata": {},
+   "source": [
+    "### 2. Create a Customized GraphStorm-compatible RGCN Model for the Time Series Node Regression\n",
+    "\n",
+    "GraphStorm has a set of GNN component modules that could be freely combined for different tasks. This notebook depends on an RGCN model, `RgcnNRModel4TS`, that extends from the `GSgnnModel` and implements the sliding window method in its `forward()` and `predict()` function for node regression by using GaphStorm APIs. Users can find the details of the `RgcnNRModel4TS` in the `nr_models.py` file.\n",
+    "\n",
+    "Because our synthetic air transportation network have both static features and time series features, the customized `RgcnNRModel4TS` need to know which features are static or time series. So the `RgcnNRModel4TS` class asks this information in its arguments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "9ba68c6f-d419-4f39-a3c4-340b256a83c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nr_models import RgcnNRModel4TS\n",
+    "\n",
+    "# initialize the model with specific time series related arugments\n",
+    "ts_nfeat_names = {'airport':['inventory_amounts']}\n",
+    "ts_efeat_names = {('airport', 'demand', 'airport'): ['demands'], \\\n",
+    "                  ('airport', 'traffic', 'airport'): ['traffics']}\n",
+    "\n",
+    "# import a customized RGCN model for node time series regression\n",
+    "model = RgcnNRModel4TS(g=ml_data.g,\n",
+    "                       num_hid_layers=len(fanout),\n",
+    "                       node_feat_field=nfeats_4_modeling,\n",
+    "                       edge_feat_field=efeats_4_modeling,\n",
+    "                       edge_feat_mp_op='add',\n",
+    "                       target_ntype='airport',\n",
+    "                       ts_nfeat_names=ts_nfeat_names,\n",
+    "                       ts_efeat_names=ts_efeat_names,\n",
+    "                       hid_size=128,\n",
+    "                       ts_size=NUM_DEMAND_DAYS,\n",
+    "                       window_size=7)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50883e00-ae86-4773-aa81-935fba9e72b6",
+   "metadata": {},
+   "source": [
+    "### 3. Setup a GraphStorm Evaluator\n",
+    "\n",
+    "To check the performance during model training, GraphStorm relies on a set of built-in `Evaluator`s for different tasks. Here we create a `GSgnnRegressionEvaluator` for the node time series regression task."
```

Comment:
Is GSgnnRegressionEvaluator built-in evaluator?
Can we say `graphstorm.eval.GSgnnRegressionEvaluator` here?

Reply:
Yes, it is. Modified.